Makefile

```diff
@@ -252,4 +252,4 @@ version-docs:
 		-w /docs \
 		-u $(shell id -u):$(shell id -g) \
 		node:${NODE_VERSION} \
-		sh -c "yarn install --frozen-lockfile && yarn run docusaurus docs:version ${NEWVERSION}"
\ No newline at end of file
+		sh -c "yarn install && yarn run docusaurus docs:version ${NEWVERSION}"
```

Comment:
why did we remove `--frozen-lockfile`?

Reply:
I was getting this error to update the lockfile: "error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`."